spec/features/forms/form_item/constraints_form_spec.rb

```diff
@@ -58,6 +61,10 @@
             select_operator("< less than")
             fill_in_value("25")
           end
+          click_link("Add Rejection Message")
+          wait_modal_to_be_visible
```

Comment:
Let's wait on diagnosing this spec stuff until we are feeling good about the stuff above.